extensions-core/kafka-indexing-service/src/main/java/org/apache/druid/indexing/kafka/KafkaConsumerMonitor.java

```diff
@@ -172,6 +176,13 @@ public boolean doMonitor(final ServiceEmitter emitter)
           emitter.emit(builder.setMetric(kafkaConsumerMetric.getDruidMetricName(), emitValue));
         }
       }
+
+      // Capture `poll-idle-ratio-avg` metric for autoscaler purposes.
+      if (POLL_IDLE_RATION_METRIC_NAME.equals(metricName.name())) {
```

Comment:
Since we are capturing it anyway, let's emit it too. Might be useful for debugging purposes.

Reply:
I slept with that idea - it deserves a separate patch IMO.